translations/combat.tex/pl.po

```diff
@@ -516,6 +739,14 @@ msgid ""
 "The previously played Attack and Defense cards no longer have any effect.}\n"
 "\n"
 msgstr ""
+"\\textit{Po zagraniu wszystkich kart dla danej sekwencji ataku wykonywany jest rzut Kością Ataku celem policzenia jego całkowitej siły .\n"
+"Bob wyrzuca +1.\n"
+"Zwiększa to Atak Zombie z 3 na 4, co zmniejszone zostaje o Obronę Gryfów o 1. Gryfy otrzymują 3 \\includesvg[height=10px]{\\svgs/damage.svg}. Ponieważ ich PŻ \\includesvg[height=10px]{\\svgs/health_points.svg} wynosi 4, ich karta nie zostaje obrócona na stronę \"Kilka\" .}\\par\n"
+"\\textit{Na Gryfach nie ma położonej Czarnej Kostki, więc wykonują one Kontratak.\n"
+"Normalnie Kostka powinna się na nich znajdować, jednakże ich Specjalna \\includesvg[height=10px]{\\svgs/unit_retaliate.svg} zdolność umożliwia wykonanie Kontrataku nieograniczoną ilość razy, więc Kostki nie ustawia się.}\\par\n"
+"\\textit{Obaj Gracze ponownie mogą zmodyfikować Statystyki swoich Jednostek podczas Kontrataku.\n"
+"Poprzednio zagrane karty Ataku i Obrony nie mają już żadnego efektu.}\n"
```

Comment:
widzę tu przełożenie z angielskiego "no longer have any effect" ale nie wiem czy to dobrze brzmi po polsku..

```suggestion
"Poprzednio zagrane karty Ataku i Obrony nie wliczają się już do tego ataku.}\n"
```